mLRS/CommonRx/mlrs-rx.cpp

```diff
@@ -907,3 +914,5 @@ dbg.puts(s8toBCD_s(stats.last_rssi2));*/
   }//end of while(1) loop
 }//end of main
 #endif
+
+void setup(void) {esp_setup();};
```

Comment:
why that?

Reply:
The ESP.restart() is a clumsy way just to reload the saved params, so I wanted to avoid it with this. I want to bring in your setup()/loop() solution soon with some testing. This branch is not fully stm32 compatible at this point.

Reply:
as far as I can see this code construct isn't doing anything
is it in anticipation of esp32? If so, don't do here yet.

the setup()/loop() apporach needs massive testing on all plöatforms and all targets and in all configurations before it can be brought into anything which may bee seen stable. Shouldn't be ripe for main.